pkg/framework/aries/framework.go

```diff
@@ -208,7 +211,8 @@ func (a *Aries) Context() (*context.Provider, error) {
 		context.WithInboundTransportEndpoint(a.inboundTransport.Endpoint()),
 		context.WithStorageProvider(a.storeProvider),
 		context.WithTransientStorageProvider(a.transientStoreProvider),
-		context.WithCrypter(a.crypter),
+		context.WithDefaultPacker(a.packer),
```

Comment:
Should I assume that `a.packer` must also be in the `a.packers` list in L215?

Reply:
Packager handles that - if the default packer is not in the list, it's added.

Reply:
Specifically - each Packer defines an encoding type string (returned by `Packer.EncodingType()`) which is used to key into the packer map. So if `a.packer`'s type is the type of one of the elements of `a.packers`, then it's assumed to be in the list already.